tests/testthat/_snaps/shuffle_ard.md

```diff
@@ -16,378 +16,26 @@
 ---
 
     Code
-      ard_shuffled
```

Comment:
@bzkrouse Largely, I don't think there were substantive changes to the shuffle snapshots. The order of the rows is different, however. 

I made the snapshot shorter by only printing a handful of rows. Is that ok? If we do need the full snapshot, can we split it into multiple smaller snapshots?